test/Indice.Cryptography.Tests/HttpTokenValidationTests.cs

```diff
@@ -63,4 +63,41 @@ public void HttpTokenValidationTest() {
         Assert.True(disgestIsValid);
         Assert.True(signatureIsValid);
     }
+
+    [Fact]
+    public void HttpTokenValidationTest_HMAC() {
+        // System.ArgumentOutOfRangeException: 'IDX10720: Unable to create KeyedHashAlgorithm for algorithm
+        // 'http://www.w3.org/2001/04/xmldsig-more#hmac-sha256', the key size must be greater than: '256' bits,
+        // key has '160' bits. (Parameter 'keyBytes')'
+
+        // for the above reason, we are setting a GUID as key 
+        var privateKey = "379af4ab-d390-410a-8da1-802d85d59eb5";
```

Comment:
not good actually we will need to pass the actual secret phrase though a hashing algorithm before using that as a HMAC256 key. This is essential because we cannot predict the input key size for symmetric key signing.